src/components/Screen/index.js

```diff
@@ -74,26 +75,38 @@ export class Screen extends Component {
 				</Header.Content>
 				<Header.Actions>
 					<Header.Action
-						title={notificationsEnabled ? 'Disable notifications' : 'Enable notifications'}
+						title={notificationsEnabled ? I18n.t('Disable notifications') : I18n.t('Enable notifications')}
```

Comment:
@tassoevan, after adding the new `Tooltip component` on `Header.Actions`, we should remove the `title` property, right?